pwndbg/commands/plist.py

```diff
@@ -189,17 +188,17 @@
 @pwndbg.commands.ArgparsedCommand(parser, command_name="plist")
 def plist(
     path: str,
-    next: int,
+    next: str,
     sentinel: int,
-    inner_name: str,
-    field_name: str,
+    inner_name: Optional[str],
+    field_name: Optional[str],
```

Comment:
Why typing changes here?

Reply:
next is string
```
parser.add_argument(
    "next", type=str, help="The name of the field pointing to the next element in the list"
)
```

inner_name, field_name - they have default=None
```
parser.add_argument(
    "-f",
    "--field",
    dest="field_name",
    type=str,
    # missing is: default=None
    help="The name of the field to be displayed, if only one is desired",
)
```